# Checks that multiline comments are haikus (haiku-comments)

Please describe the origin of the rule here.

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js
/*
 * once upon a time
 * there was a bored JS dev
 * who could not write haiku
 */
```

Examples of **correct** code for this rule:

```js
/*
 * once upon a time
 * there was a bored JS dev
 * who could write haiku!
 */
```

### Options

If there are any options, describe them here. Otherwise, delete this section.

## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule.

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.
